python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -92,66 +92,94 @@ class NearestNeighbors(Base):
 
       import cudf
       from cuml.neighbors import NearestNeighbors
-      import numpy as np
+      from cuml.datasets import make_blobs
 
-      np_float = np.array([
-        [1,2,3], # Point 1
-        [1,2,4], # Point 2
-        [2,2,4]  # Point 3
-      ]).astype('float32')
+      X, _ = make_blobs(n_samples=25, centers=5,
+                        n_features=10, random_state=42)
 
-      gdf_float = cudf.DataFrame()
-      gdf_float['dim_0'] = np.ascontiguousarray(np_float[:,0])
-      gdf_float['dim_1'] = np.ascontiguousarray(np_float[:,1])
-      gdf_float['dim_2'] = np.ascontiguousarray(np_float[:,2])
+      # build a cudf Dataframe
+      X_cudf = cudf.DataFrame.from_gpu_matrix(X)
 
-      print('n_samples = 3, n_dims = 3')
-      print(gdf_float)
+      # fit model
+      model = NearestNeighbors(n_neighbors=3)
+      model.fit(X)
 
-      nn_float = NearestNeighbors()
-      nn_float.fit(gdf_float)
       # get 3 nearest neighbors
-      distances,indices = nn_float.kneighbors(gdf_float,k=3)
+      distances, indices = model.kneighbors(X_cudf)
 
+      # print results
       print(indices)
       print(distances)
 
     Output:
 
     .. code-block:: python
 
-      import cudf
-
-      # Both import methods supported
-      # from cuml.neighbors import NearestNeighbors
-      from cuml import NearestNeighbors
-
-      n_samples = 3, n_dims = 3
-
-      dim_0 dim_1 dim_2
-
-      0   1.0   2.0   3.0
-      1   1.0   2.0   4.0
-      2   2.0   2.0   4.0
 
-      # indices:
+    indices:
+
+             0   1   2
```

Comment:
This is a pretty long comment... do you want to truncate the output or add an ellipsis?